packages/big-design/src/components/Button/Button.tsx

```diff
@@ -19,26 +19,18 @@ export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement>, Ma
   variant?: 'primary' | 'secondary' | 'subtle';
 }
 
-const RawButton: React.FC<ButtonProps & PrivateProps> = memo(({ forwardedRef, ...props }) => {
-  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
No need to have a custom click handler. The disabling happens in the browser automatically